Jenkinsfile

```diff
@@ -146,6 +146,11 @@ pipeline {
                                 sh "npm run build -- ${BUILD_TYPE} --channel=${CHANNEL} --target_os=android --target_arch=arm"
                             }
                         }
+                        stage("dist") {
+                            steps {
+                                sh "npm run create_dist -- ${BUILD_TYPE} --channel=${CHANNEL} --target_os=android --target_arch=arm"
```

Comment:
I think we also need `android_override_version_name` here and build - https://github.com/brave/brave-browser/pull/7828/files#diff-1bb8ffaf71730e8067757db1a6a153dbR76

Reply:
@SergeyZhukovsky can provide more details about the value for it

Reply:
we still need to provide `android_override_version_name` that is taken from `package.json` desktop file I think. `android_override_version_name` is only a command line option

Reply:
we fixed it on brave-browser side